packages/trader/src/App/app.tsx

```diff
@@ -26,24 +25,11 @@ const TradeModals = Loadable({
 
 const App = ({ passthrough }: Apptypes) => {
     const root_store = initStore(passthrough.root_store, passthrough.WS);
-    const { isTabletPortrait } = useDevice();
+
     React.useEffect(() => {
         return () => root_store.ui.setPromptHandler(false);
     }, [root_store]);
 
-    React.useEffect(() => {
-        const landscapeBlockerElement = document.querySelector('.landscape-blocker');
-        if (landscapeBlockerElement && !isTabletPortrait) {
-            landscapeBlockerElement.classList.add('landscape-blocker--hidden');
-        }
-
-        return () => {
-            if (landscapeBlockerElement) {
-                landscapeBlockerElement.classList.remove('landscape-blocker--hidden');
-            }
-        };
-    }, [isTabletPortrait]);
-
```

Comment:
Hi @sergei-deriv , why did you remove this? Did you the logic somewhere else?

Reply:
@farhan-nurzi-deriv Yes, it's because I have blocker component [here](https://github.com/binary-com/deriv-app/pull/13396/files#diff-532239524d47017e756b81d37404d33525998cdeb3dc06bb301787b6e81e6799), and handling disabled routes [here](https://github.com/binary-com/deriv-app/pull/13396/files#diff-eb98195e91448784cde931aeb8ae8b55dda93ee719b01f2df8f61ccc8febbbf6)

Reply:
The logic is different for dtrader route. We need to show the blocker if its not real tablet device. Please keep it and i will update it in my PR 🙏

Reply:
@farhan-nurzi-deriv show me please your 2nd PR

Reply:
I will just follow your implementation to avoid conflicts https://github.com/binary-com/deriv-app/pull/14781

Reply:
@farhan-nurzi-deriv good job, I've checked your PR and it looks fine except one thing (already left a comment). Nice we are finally in sync)